com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/ServiceStep.java

```diff
@@ -66,16 +70,20 @@ protected String getStepErrorMessage(DelegateExecution context) {
     }
 
     protected abstract MethodExecution<String> executeOperation(DelegateExecution context, CloudControllerClient controllerClient,
-        CloudServiceExtended service) throws Exception;
-    
+                                                                CloudServiceExtended service);
+
     protected abstract ServiceOperationType getOperationType();
-    
+
     public ServiceUpdater getServiceUpdater() {
         return serviceUpdater;
     }
-    
-    public ServiceGetter getServiceInstanceGetter() {
-        return serviceInstanceGetter;
+
+    public ServiceOperationGetter getServiceOperationGetter() {
```

Comment:
Is this used outside the hierarchy ?

Reply:
No. The methods will be changed to protected.